### ISSUE TITLE
`suite new` command throws error

### ISSUE THREAD

**Issue by colmsnowplow** [fivetran/great_expectations#1325]
Autogeneration of a new suite for a Redshift table threw the following error:

```
Building Data Docs...
--- Logging error ---
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/great_expectations/render/renderer/site_builder.py", line 368, in build
    rendered_content = self.renderer_class.render(resource)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/great_expectations/render/renderer/page_renderer.py", line 69, in render
    if column not in columns:
TypeError: unhashable type: 'list'
```

Eventually the docs (and the test suite) were generated just fine, so not a blocking issue.

**Comment by Aylr** [comment]
@colmsnowplow Which driver are you using? @roblim does this ring any bells for you?

**Comment by colmsnowplow** [comment]
I _think_ it was snowflake. However I just started afresh in a new project to double check that, and I haven't run into it again. (BQ, Redshift and Snowflake suites all generated without hitting this again).

Over the weekend I resolved a gnarly problem I was having with python2 vs python3 on my classpath (for unrelated reasons) - so it is entirely possible that this issue is between keyboard and chair :D. My bad, apologies for the noise.
